src/pages/signup.js

```diff
@@ -48,12 +51,24 @@ const Signup = () => {
     try {
       await signup(data).unwrap();
       toast.success("Account created");
-      navigate("/login");
+      
     } catch (error) {
       // we handle errors with middleware
     }
   }
 
+  async function handleShow(data) {
```

Comment:
Hi @steph237 great work. Here is how you can make it better.

Instead of creating a new function that also does signup, you can replace the navigate("/login")   in the handleSignup function with setOpen(true)